runtime/src/iree/tooling/parameter_util.c

```diff
@@ -20,11 +20,12 @@
 
 IREE_FLAG(
     string, parameter_mode, "mmap",
```

Comment:
Not sure if this is just a quirk of how multiple PRs stack in github, but we add "file" to the description, but do not check for it in `iree_io_open_parameter_file`. However I know one of the stacked PRs does it.

Reply:
ah yeah that slipped through - I'll fix up while merging so history is nice